src/Sylius/Component/Support/Adapter/ConfigNotificationRecipientAdapter.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Lakion package.
+ *
+ * (c) Lakion
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+namespace Sylius\Component\Support\Adapter;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+class ConfigNotificationRecipientAdapter implements ConfigNotificationRecipientAdapterInterface
```

Comment:
And should be named: ArrayRecipientsAdapter. I'm thinking that adapter pattern is a bit overkill here, ArrayRecipientsProvider would be enough. WDYT?